### PR TITLE
Octane - Replace some references to Computed Properties with Tracked

### PR DESCRIPTION
**Pull request by jenweber**
This is incremental work towards #669

### CODE REVIEW

File: guides/release/applications/dependency-injection.md [hunk 1]
```diff
@@ -234,16 +234,18 @@ For example, this component plays songs with different audio services based
...
 
 // Usage:
 //
-// {{play-audio song=song}}
+// <PlayAudio @song=this.song />
 //
-export default Component.extend({
-  audioService: computed('song.audioType', function() {
+export default class PlayAudioComponent extends Component {
+  @tracked song;
```
Comment (rwjblue) [inline guides/release/applications/dependency-injection.md]: I wouldn't use `@tracked` at all here (once migrated to `@glimmer/component`), I'd probably do:

```js
export default class PlayAudioComponent extends Component {
  get audioService() {
    if (!this.args.song) {
      return null;
    }

    let applicationInstance = getOwner(this);
    let { audioType } = this.args.song;

    return applicationInstance.lookup(`service:audio-${audioType}`);
  }

  click() {
    let player = this.audioService;
    player.play(this.args.song.file);
  }
}
```

File: guides/release/applications/dependency-injection.md [hunk 2]
```diff
@@ -252,11 +254,11 @@ export default Component.extend({
     let audioType = this.song.audioType;
 
     return applicationInstance.lookup(`service:audio-${audioType}`);
-  }),
+  };
```
Comment (rwjblue) [inline guides/release/applications/dependency-injection.md]: I find it fairly odd to have semicolons here (I had never seen this before in class bodies until recently), I would normally not put anything here (no semicolon, no comma, just a new line).

I don't think this is a giant issue, but we do need to ensure that the guides consistently follow one pattern or the other (either semicolons or not).

File: guides/release/controllers/index.md [hunk 3]
```diff
@@ -103,5 +103,5 @@ Yes! Controllers are still an integral part of an Ember application architecture
 #### When should we create a Controller?
 
 * We want to pass down actions or variables to share with a Route’s child components
-* We have a computed property that depends on the results of the model hook
+* We have a getter that needs the results of the model hook
```
Comment (rwjblue) [inline guides/release/controllers/index.md]: I think that just saying "getter" here is a tad odd, we **are** computing a value (just not a `Ember.computed`), what do yo uthink about:

```suggestion
* We need to compute a value based on the results of the model hook
```

File: guides/release/models/relationships.md [hunk 4]
```diff
@@ -267,7 +267,7 @@ updated and saved this often results in the creation of a great deal
 of code for very little benefit. An alternate approach is to define
 these relationships using an attribute with no transform
 (`attr()`). This makes it easy to access readonly values in
-computed properties and templates without the overhead of defining
+getters and templates without the overhead of defining
```
Comment (rwjblue) [inline guides/release/models/relationships.md]:
```suggestion
other objects and templates without the overhead of defining
```

File: guides/release/routing/query-params.md [hunk 5]
```diff
@@ -41,15 +41,14 @@ and the `category` property on `controller:articles`. In other words,
 once the `articles` route has been entered, any changes to the
 `category` query param in the URL will update the `category` property
 on `controller:articles`, and vice versa.
-Note that you can't bind `queryParams` to computed properties, they
+Note that you can't make `queryParams` be a getter; they
```
Comment (rwjblue) [inline guides/release/routing/query-params.md]:
```suggestion
Note that you can't make `queryParams` be a dynamically generated property (neither computed property, nor property getter); they
```

File: guides/release/applications/dependency-injection.md (hunk 2 above)
Comment (jenweber) [reply guides/release/applications/dependency-injection.md]: We're using a mix everywhere now :( I'll make a note to catch it in an editing pass.

**Pull request merged**